doc/api/deprecations.md

```diff
@@ -2651,6 +2651,19 @@ written twice. This introduces a race condition between threads, and is a
 potential security vulnerability. There is no safe, cross-platform alternative
 API.
 
+<a id="DEPXXX"></a>
+### DEPXXX: Use `request.destroy()` in favor of `request.abort()`
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/32807
+    description: Documentation-only deprecation.
+-->
+
+Type: Documentation-only
+
+Use [`request.destroy()`][] in favor of [`request.abort()`][].
```

Comment:
"in favor of" usually means that the thing after the phrase is the thing to use. Maybe "in place of" or "instead of"?